src/utilities/read_geo_factors.cxx

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
do we need this file once the `BinNormalisation` is sorted?  If we do (or you want to keep it), rename it to `read_GEHEAD5_geo_factors`

Reply:
No this file won't be there once it is shifted to BinNormalisationFromGEHDF5